application/common/models/User.php

```diff
@@ -59,6 +59,9 @@ public function fields()
             'last_name',
             'idp_username',
             'email',
+            'password' => function ($model) {
```

Comment:
Would it be good to call this `passwordMeta` instead of `password` (to avoid using the name "password" to refer to yet another thing)?